doc-General_Configuration/topics/Configuration.adoc

```diff
@@ -2797,9 +2799,10 @@ The region number must be unique on each {product-title} instance where replicat
 [[configuring-a-remote-copy]]
 ===== Configuring a Remote Copy
 
-Configure a {product-title} instance to act as a remote copy where data from the global copy will be replicated. This procedure must only be performed on instances where the *Database Synchronization* role will be enabled. This procedure must be performed before enabling that role.
+Configure a {product-title} instance to act as a remote copy where data from the global copy will be replicated.
```

Comment:
This is a bit ambiguous. It almost makes it seem like the data is coming *from* the global region.

Maybe:
> Configure a {product-title} instance to act as a remote copy from which data will be replicated to the global copy.